x-pack/plugins/osquery/cypress/e2e/all/alerts_multiple_agents.cy.ts

```diff
@@ -43,12 +44,14 @@ describe(
     it('should substitute parameters in investigation guide', () => {
       cy.getBySel('expand-event').first().click();
       cy.getBySel('securitySolutionFlyoutInvestigationGuideButton').click();
-      cy.contains('Get processes').click();
-      cy.getBySel('flyout-body-osquery').within(() => {
-        cy.contains("SELECT * FROM os_version where name='Ubuntu';");
-        cy.contains('host.os.platform');
-        cy.contains('platform');
-      });
+      // Flakes at times if the button is only clicked once
```

Comment:
@patrykkopycinski  & @tomsonpl  fyi .. I made some changes to your cypress tests here